### PR TITLE
Adding justgo 1.0.1

### PR DESCRIPTION
**Pull request by inadarei**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x ] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/justgo.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'justgo' do
+  version '1.0.1'
+  sha256 '74864b00222d1e679d3673ada2d31e5c6ea8af6ca5572e42293481a862c193c6'
+
+  url 'https://github.com/inadarei/justgo/blob/master/cmd/justgo/build/justgo.zip?raw=true'
+  name 'justgo'
+  homepage 'https://github.com/inadarei/justgo'
+
+  app 'justgo'
```
Comment (vitorgalvao) [inline Casks/justgo.rb]: Not an `app`.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
This seems to be a CLI-only open-source tool. Please submit it first to [Homebrew/core](https://github.com/Homebrew/homebrew-core) and only if they don’t accept it, submit here (linking back to the pull request where it was refused).

This policy is documented in [faq/rejected_casks](https://github.com/caskroom/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> The app is both open-source and CLI-only (i.e. it only uses the `binary` artifact). In that case, and [in the spirit of deduplication](https://github.com/caskroom/homebrew-cask/issues/15603), submit it first to [Homebrew/core](https://github.com/Homebrew/homebrew-core). If it is rejected there, you may then try again in Homebrew-Cask (link us to the issue on Homebrew so we can see their reasoning for rejection).

**Comment by inadarei** [comment]
@vitorgalvao thank you for such prompt response. This is my first brew submission so I may not understand how to properly do it, but when I read instructions at homebrew/core it explicitely said they do *not* like binary submissions and any binary submission should be directed to Cask?

What I am submitting is a Go CLI tool and it's important the installation to be binary / not require pre-existing Go environment (otherwise a `go get` is more than sufficient).

Can you please clarify why a binary submission is redirected to core? Thanks

**Comment by vitorgalvao** [comment]
Why can’t this be built from source? That’s the point.

Unfortunately, I just noticed neither HB nor HBC will accept this. You’re **self-submitting** a tool with **too few stars**. We don’t accept those to keep the repos manageable with software other people will actually use.

**Comment by inadarei** [comment]
@vitorgalvao it can be built from source but that defeats the purpose. The only reason anybody would install a Go package with brew is to download a binary (and not have to install Go environment). If somebody already has a working Go environment, or is willing to install one, then `go get <githuburl>` is both simpler and easier.

**Pull request closed without merging**